sdk/core/azure-core/test/libcurl-stress-test/README.md

```diff
@@ -19,7 +19,7 @@ Where namespace will be created if missing , search directory can be any folder
 
 ATM the docker image is build by hand and hard-coded in the chart to simplify matters.  
 
-To build the image run "docker build -t <acr>/azuresdkforcpp/curlstress:v8  --build-arg targetTest=azure-core-libcurl-stress-test --build-arg build=on  ."
+To build the image run "docker build -t <acr>/azuresdkforcpp/curlstress:v8  --build-arg targetTest=azure-core-libcurl-stress-test --build-arg build=on  -f .\Dockerfile .\..\..\..\..\..\"
```

Comment:
Lol, jinx (i.e. ditto)

Reply:
the ..\...\..... represents the root of the tree that docker has access to , since we need to build inside the docker file it needs access to the root of the enlistment to copy it locally , in order to build the test binary and then add it to the final docker image